test/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM fedora:37
```

Comment:
Not to undo all the hard work you put in, but the more I think about NOAA's response to our containers discussion, the more I think that for now we shouldn't include a container at all, and instead just try to directly use the `ubuntu-latest` environment provided by Github, which is what they do.  This way we might have an easier time getting our framework version accepted and merged in.  Plus we'll always have our eventual CAM-SIMA container to use for our own framework development (as we will have full control over that one).